sdk/storage/storage-file-share/CHANGELOG.md

```diff
@@ -149,7 +149,7 @@
 - [Breaking] `forceCloseHandlesSegment` is not exposed from the library in favour of the new method `forceCloseAllHandles` on `FileClient` and `DirectoryClient`. [PR #5620](https://github.com/Azure/azure-sdk-for-js/pull/5620)
 - [Breaking] IE11 needs `Object.assign` polyfill loaded. [PR #5727](https://github.com/Azure/azure-sdk-for-js/pull/5727)
 
-## 12.0.0-preview.4 (2019.10)
+## 12.0.0-preview.4 (2019-10-01)
```

Comment:
Minor: 
Release date should be 2019-10-09, but  it doesn't impact sorting.

Reply:
```suggestion
## 12.0.0-preview.4 (2019-10-09)
```